.github/workflows/continuous-integration.yml

```diff
@@ -58,7 +50,39 @@ jobs:
       - name: "Run expect tests"
         run: "composer test"
 
+  code-coverage:
+    needs: test
+    if: github.repository == 'ramsey/composer-install'
```

Comment:
Nice! I didn't know you could do this to limit certain jobs to a specific repository.